external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSinkSuite.scala

```diff
@@ -160,76 +204,93 @@ class KafkaSinkSuite extends StreamTest with SharedSQLContext with KafkaTest {
     }
   }
 
-  test("streaming - write aggregation w/o topic field, with topic option") {
+  /**
```

Comment:
Not sure this description needed. The code should reflect the intention and add comment if it's tricky (I know it's from the legacy code).
Such comments can deviate from code (where is 'bar' for example) and can cause confusion.